poetry/masonry/builders/sdist.py

```diff
@@ -137,6 +137,11 @@ def build_setup(self):  # type: () -> bytes
             else:
                 pass
 
+        license_files = list(map(str, self._package.license_files))
```

Comment:
why the `map` when the schema defines an item to be a string?

Reply:
The map is to transform them into strings. Since `self._package.license_files` is a list of `Path` objects they were getting written out to the final `setup.py` file as `repr()` style objects `<Path ... >` which was't very useful for setuptools. :-)